frontend/src/reports/ReportProvider.tsx

```diff
@@ -190,10 +191,10 @@ function ReportProvider(props: { madLib: MadLib; setMadLib: Function }) {
           Until then, the following states appear grey when viewing COVID-19
           maps featuring hospitalizations and deaths: <b>Hawaii</b>,{" "}
           <b>Nebraska</b>, <b>South Dakota</b>, and <b>Wyoming</b>.{" "}
-          <b>Delaware</b> and <b>West Virginia</b> are included when viewing
-          hospitalizations but appear as grey when viewing reports on deaths.{" "}
-          <b>Rhode Island</b> appears as grey when viewing reports on
-          hospitalizations but is included when viewing deaths.
+          <b>Delaware</b>. Is included when viewing hospitalizations but appear
```

Comment:
remove the `.` and lowercase `Is` i think?

Reply:
and should be `appears` now since it's a single state

Reply:
thx

Reply:
I'd keep the sentence structure consistent I think it should be:

```
<b>Delaware</b> is included when viewing hospitalizations, but appears as grey when viewing reports on deaths.
<b>Rhode Island</b> is included when viewing deaths, but appears as grey when viewing reports on hospitalizations.

Also, I feel like all of these `appears as grey` should be `appears grey`.... right?

Reply:
I guess, I don't know